.github/ISSUE_TEMPLATE/build.yml

```diff
@@ -0,0 +1,75 @@
+name: Build Issue Report
+description: This report is for the build/installation issue
+title: "[Build]: "
+labels: ["build", "support_request"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please provide all the necessary information to expedite the response.
+  - type: input
+    id: ov_version
+    attributes:
+      label: OpenVINO Version 
+      description: OpenVINO version, branch, or tag in OpenVINO GitHub
+      placeholder: 2021.4.0 LTS / Master Branch / tag 2022.3.0
+    validations:
+      required: true
+  - type: dropdown
+    id: os
+    attributes:
+      label: Operating System
+      description: What OS are you using?
+      options:
+        - Ubuntu 18.04 (LTS)
+        - Ubuntu 20.04 (LTS)
```

Comment:
Let's add:
- Ubuntu 22.04
- openSUSE

We can also add a dedicated field "architecture" with values "x86 (64 bits) x86 (32 bits), arm (64 bits), arm (32 bits), risc-v"

Reply:
I agree with @ilya-lavrenov. @hbalasu1 can you please edit?

Reply:
Done added